spec/controllers/users_controller_spec.rb

```diff
@@ -39,6 +39,24 @@
       end
     end
   end
+
+  describe 'GET show' do
+    it 'sets @user' do
+      set_current_user
+      get :show, id: Fabricate(:user).id
+      expect(assigns(:user)).to eq(User.second)
```

Comment:
A more explicit way to do this would be:

```ruby
it 'sets @user' do
  set_current_user
  alice = Fabricate(:user)
  get :show, id: alice.id
  expect(assigns(:user)).to eq(alice)
end
```

This gives a higher level on confidence in the spec since it is more explicit.